Makefile

```diff
@@ -55,8 +55,13 @@ e2e:
 	yarn build && \
 	yarn test:e2e
 
-e2e-in-ci:
-	. ./.envrc && \
+e2e-in-container:
```

Comment:
don't like the distinction `-in-container` vs `in-docker` ... need more explicit names to differentiate.